<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherMainForm</class>
 <widget class="QMainWindow" name="TeacherMainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>出卷系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="m_listViewProblem">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>21</y>
      <width>201</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="m_widgetProblem" native="true">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>19</y>
      <width>501</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_buttonAdd">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_buttonMinus">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_buttonFinish">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>完成</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="m_actionOpenFile"/>
    <addaction name="m_actionSaveFile"/>
    <addaction name="separator"/>
    <addaction name="m_actionExit"/>
   </widget>
   <addaction name="fileMenu"/>
  </widget>
  <action name="m_actionSaveFile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="m_actionExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="m_actionOpenFile">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_actionExit</sender>
   <signal>triggered()</signal>
   <receiver>TeacherMainForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
